src/vocgui/static/scripts.js

```diff
@@ -90,6 +90,10 @@ function next_document() {
       return;
     } else if (word_status == 2 && !wrong_answer) { 
     	documents_almost_correct = documents_almost_correct + 1;
+    	$("#input_word").addClass("almost_valid");
+    	$("#input_word").val(old_document["fields"]["word"]);
+    	documents_correct = documents_correct - 1;
```

Comment:
Instead of reducing correct words by one, it would be better to set `wrong_answer = true;` as in line 89. This prevents `documents_correct` from incrementing, see line 94.